### ISSUE TITLE
Error in the run_segmentation.py file.

### ISSUE THREAD

**Issue by sjg02122** [isl-org/DPT#36]
I have a question for your run_segmentation.py.
Looking at the structure of the DPT model, the foward returns five outputs.

[out, layer1, layer2, layer3, layer4]

In run_segmentation.py, insert a sample into the model and enter the above-mentioned type of list as out.
After that, Torch.nn.functional.If you put it in the interpolate, you will encounter the following error.

AttributeError: 'list' object has no attribute 'dim'

How do you solve it?

**Comment by ranftlr** [comment]
I don't think we ever output such a list of 5 outputs. Can you point me to the exact line where this happens?

**Comment by sjg02122** [comment]
I'm sorry that just my mistake.

Another question is the performance.

I downloaded the weight "dpt_hybrid-ade20k-53898607.pt"

and I ran the code run_segmentation.py and I got the below result.
I think this result has a gap with paper.

How can I get the better result?

Result
![000003_10_r](https://user-images.githubusercontent.com/33753741/124617523-f40cf680-deb1-11eb-8139-34276fed21fe.png)

Original
![000003_10](https://user-images.githubusercontent.com/33753741/124617540-f8391400-deb1-11eb-9ba2-0867af36438a.png)
